README.md

```diff
@@ -12,6 +14,7 @@ SESSION_TOKENS_DURATION_SECONDS=1800
 TENANTS_IDS_FILE_PATH="./data/dev/tenants-ids.json"
 MAX_POLLING_TRIES=32
 POLLING_SLEEP_TIME=100
+CUCUMBER_OPTS_PARALLEL=5
```

Comment:
This should be added in the env schema in the `env.ts` file

Reply:
env var added to env schema